templates/post.html

```diff
@@ -121,7 +121,12 @@ <h1 class="post_title">
 			<div class="post_footer">
 				<ul id="post_links">
 					<li><a href="{{ post.permalink }}">permalink</a></li>
-					<li><a href="https://reddit.com{{ post.permalink }}" rel="nofollow">reddit</a></li>
+					<li>
+						<a id="reddit_link" {% if prefs.visit_reddit_confirmation != "on" %}href="#popup"{% else %}href="https://reddit.com{{ post.permalink }}" rel="nofollow"{{ url }}"{% endif %}>reddit</a>
+						{% if prefs.visit_reddit_confirmation != "on" %}
```

Comment:
@Daniel-Valentine: This is going to cause a conflict with #637. Make sure to port these changes into `src/utils.html` in the branch in #637.